maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,40 @@
+.card {
+  background: $green;
+  border-radius: 5px;
+  height: 270px;
+  width: 180px;
+
+  &::before {
+    border: solid 1px $green;
+    border-radius: 5px;
+    content: '';
+    height: 270px;
+    margin: -7px;
+    position: absolute;
+    width: 180px;
+  }
+}
+
+.tech {
+  position: relative;
+  h4 {
```

Comment:
Remember that isn't a good practice this nesting and to set styles a general elements